### ISSUE TITLE
Option to mark as read

### ISSUE THREAD

**Issue by KSXGitHub** [shumingch/gnome-email-notifications#15]
It is useful when mail content is short (e.g. Travis CI build status).

**Comment by brian-provenzano** [comment]
Agreed - for these types of emails this would be a great convenience

**Comment by brian-provenzano** [comment]
OK - I took a quick look at the code and I can see that this feature would probably take a little time to implement.  It appears that it is currently using the gmail atom api to get all unread messages in the inbox and AFAIK (or can tell from google's docs) this API  read-only.  I'm guessing you would have to switch to the full REST API that enables modifications (e.g. marking messages as read)?

Anyway -  if it can be done that would be great, but either way thank you for writing this as even in its current form it is a nice extension. :)
